likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/poll/adapter/databinder/LMFeedItemPollOptionsViewDataBinder.kt

```diff
@@ -3,16 +3,18 @@ package com.likeminds.feed.android.core.ui.widgets.poll.adapter.databinder
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import com.likeminds.feed.android.core.databinding.LmFeedItemPollOptionsBinding
-import com.likeminds.feed.android.core.poll.model.LMFeedPollOptionViewData
+import com.likeminds.feed.android.core.poll.result.model.LMFeedPollOptionViewData
 import com.likeminds.feed.android.core.ui.widgets.poll.adapter.LMFeedPollOptionsAdapterListener
+import com.likeminds.feed.android.core.ui.widgets.poll.style.LMFeedPostPollOptionViewStyle
 import com.likeminds.feed.android.core.ui.widgets.poll.view.LMFeedPollOptionView
 import com.likeminds.feed.android.core.utils.LMFeedStyleTransformer
 import com.likeminds.feed.android.core.utils.base.LMFeedViewDataBinder
 import com.likeminds.feed.android.core.utils.base.model.ITEM_POST_POLL_OPTIONS
 
 class LMFeedItemPollOptionsViewDataBinder(
     private val pollPosition: Int,
-    private val listener: LMFeedPollOptionsAdapterListener
+    private val optionStyle: LMFeedPostPollOptionViewStyle?,
+    private val listener: LMFeedPollOptionsAdapterListener?
```

Comment:
why nullable?

Reply:
Because while creating/editing we don't need any callbacks, thats why nullable. @sidcr7-likeminds